lib/api/controllers/security.js

```diff
@@ -1010,7 +1010,7 @@ class SecurityController extends NativeController {
         updatedUser,
         options);
 
-    this.kuzzle.log.info(`[SECURITY] User "${request.context.user._id}" applied action "${request.input.action}" on user "${createdUser._id}."`);
+    this.kuzzle.log.info(`[SECURITY] User "${this.getUserId(request)}" applied action "${request.input.action}" on user "${createdUser._id}."`);
```

Comment:
I know this isn't your code, but the metadata object built by this function (l:983) should use the new, safer method.

Same goes for the `updateMetadata` method